techniques/css/C35.html

```diff
@@ -110,51 +94,47 @@ <h3>Example 2: A box which expands with the text size</h3><pre xml:space="preser
    &lt;li&gt;&lt;a href="/contact/"&gt;Contact&lt;/a&gt;&lt;/li&gt;
   &lt;ul&gt;
  &lt;/nav&gt;</code>
-</pre><p>In the case of variable-width text containers for each item, the parent item may need
-                  to allow for wrapping of the items.
+</pre><p>各アイテムが可変幅テキストコンテナの場合、親アイテムは各アイテムを折り返しできるようにする必要があってもよい。
                </p>
-               <p class="working-example"><a href="../../working-examples/css-text-spacing/">Working examples of #1 and #2</a> are available.
+               <p class="working-example"><a href="../../working-examples/css-text-spacing/">事例1 と 事例２ の実例</a> を見られる。
```

Comment:
```suggestion
               <p class="working-example"><a href="https://www.w3.org/WAI/WCAG21/working-examples/css-text-spacing/">事例 1 と 事例 2 の実例</a> が利用可能である。
```

working-examplesは本家に一旦向けておきました。